spec/spec_helper.rb

```diff
@@ -1,8 +1,7 @@
 require 'rspec/support/spec'
-require 'rspec/support/ruby_features'
 
 RSpec::Support::Spec.setup_simplecov do
-  minimum_coverage 93
+  minimum_coverage 90
```

Comment:
:(

Reply:
Pretty significant drop, I agree. And it's just because of one file:
![image](https://user-images.githubusercontent.com/6916/101804847-16f54900-3b23-11eb-93f3-e78931b7a7d5.png)

This might be coincidental. [`simplecov` 0.20.0 has been released a couple of weeks ago](https://rubygems.org/gems/simplecov/versions/0.20.0). It might have improved its detection, and our coverage % dropped.

There's a ticket somewhere Myron filed to increase the coverage, surely worth addressing.